bioconda_utils/pkg_test.py

```diff
@@ -126,11 +126,15 @@ def test_package(
     tests = get_tests(path)
     logger.debug('Tests to run: %s', tests)
 
+    target = 'biocontainers'
+    if os.environ.get('QUAY_TARGET') is not None:
```

Comment:
Can we call it CONTAINER_NAMESPACE or something?

Reply:
We can name it whatever you like . ;-)

Reply:
Changed it.